### DISCUSSION TITLE
Setting font weight with -V mainfont has no effect

### DISCUSSION THREAD

**Discussion by slmjkdbtl** [jgm/pandoc#10422] [Q&A]
I'm trying to convert epub to pdf that contains CJK characters and need to use a font that contains the characters. I'm using Noto Sans SC

```
pandoc -f epub -t pdf --pdf-engine=xelatex "book.epub" -o "book.pdf" -V mainfont="Noto Sans SC-Bold" --verbose
```

The font is successfully set however specifying the font weight has no effect. In this case `Noto Sans SC-Bold` is found but the output pdf file still uses the default weight which is Light. Is this a known limitation or I don't have the correct command?

**Comment by jgm** [comment]
Not sure. But I see fontspec examples like
```
\usepackage{fontspec}
\setmainfont[
    Ligatures=TeX,
    UprightFont = *-Boo,
    ItalicFont = *-BooObl,
    SmallCapsFont = *SC-Boo,
    BoldFont = *-Dem,
    BoldItalicFont = *-DemObl
]{Futura}
```
So perhaps you could try `mainfont="Noto Sans"` and `mainfontoptions"UprightFont = * SC-Bold"` ?
